service/history/ndc/conflict_resolver.go

```diff
@@ -180,10 +180,10 @@ func (r *ConflictResolverImpl) rebuild(
 		return nil, err
 	}
 	rebuildMutableState.GetExecutionInfo().VersionHistories = versionHistories
+	rebuildMutableState.AddHistorySize(rebuiltHistorySize)
```

Comment:
this is because `rebuildMutableState` initially has history size of 0? so Add and Set are the same?

Reply:
yep